docs/source/configuration.rst

```diff
@@ -350,6 +350,79 @@ policy.
 
         *"It's some arcane magic from AT LEAST a decade ago."*
 
+Loop prevention
+---------------
+
+In order to prevent the bot from entering a loop (for example when there is
+another bot in the same channel, or if a user spams a command), it'll try to
+see if the next message to send is repeating too often in the last ten messages
+in the last few minutes. If that happens, the bot will send ``...`` a few times
+before remaining silent::
```

Comment:
```suggestion
see if the next message to send is repeating too often in a short time period.
If that happens, the bot will send ``...`` a few times before remaining silent::
```

I have been trying to think of a solution to the awkward "in the last ten messages in the last few minutes", and I think the actual solution is to just not get into those details yet. The settings to control this, described later, cover it for users who are curious without confusing someone who's just skimming the first paragraph to see what the heck this feature is.

Reply:
It's a removal then!